src/test/scala/org/broadinstitute/dsde/agora/server/dataaccess/authorization/TestAuthorizationProvider.scala

```diff
@@ -0,0 +1,24 @@
+
+package org.broadinstitute.dsde.agora.server.dataaccess.authorization
+
+import org.broadinstitute.dsde.agora.server.business.AgoraPermissions._
+import org.broadinstitute.dsde.agora.server.business.{AuthorizationProvider, AgoraAcl, AuthorizedAgoraEntity}
+import org.broadinstitute.dsde.agora.server.model.AgoraEntity
+
+import scala.collection.mutable
+
+//Test authorization implementation. Add authorizations directly to the hash map for lookup. If the entity is not found it is assumed you
+//have no permissions on that entity.
+object TestAuthorizationProvider extends AuthorizationProvider {
+  val localPermissions: mutable.Map[String, AgoraAcl] = new mutable.HashMap[String, AgoraAcl]()
+
+  def addLocalPermissions(entityId: String, authorization: AgoraAcl) = localPermissions.put(entityId, authorization)
+
+  override def authorizationsForEntity(agoraEntity: AgoraEntity): AuthorizedAgoraEntity = {
+    AuthorizedAgoraEntity(agoraEntity, localPermissions.getOrElse(agoraEntity.getUniqueIdentifier, AgoraAcl(All)))
```

Comment:
Seems like if entity is not found in localPermissions, it gets All permission. But the comment above makes it sound like it should get No permission?